<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransazioniManager</class>
 <widget class="QDialog" name="TransazioniManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="indietro_pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>108</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>Indietro</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>761</width>
     <height>210</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QDateEdit" name="inizio_dateEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>215</y>
     <width>110</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="fine_dateEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>215</y>
     <width>110</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="categoria_comboBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>145</y>
     <width>180</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="filtra_pushButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>285</y>
     <width>140</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>Conferma</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="entrate_checkBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>105</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Solo Entrate</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="uscite_checkBox">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>105</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Solo Uscite</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>65</y>
     <width>420</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Filtra Transazioni per Tipologia, Categoria e Data</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>215</y>
     <width>50</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Da:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>215</y>
     <width>50</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>A:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>145</y>
     <width>250</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Seleziona Categoria</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="salvaSuFile_pushButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>550</y>
     <width>140</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>Salva su File</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>185</y>
     <width>420</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Seleziona Periodo</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>105</y>
     <width>360</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Elimina una Transazione</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="elimina_pushButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>210</y>
     <width>110</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>Elimina</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ID_lineEdit">
   <property name="geometry">
    <rect>
     <x>595</x>
     <y>145</y>
     <width>150</width>
     <height>24</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>145</y>
     <width>150</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>ID Transazione:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="error_label">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>252</y>
     <width>420</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="errori_label_2">
   <property name="geometry">
    <rect>
     <x>455</x>
     <y>175</y>
     <width>300</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
